test/functional/bundleadd/add-multiple.bats

```diff
@@ -22,12 +22,10 @@ test_setup() {
 	assert_file_exists "$TARGETDIR/media/lib/file2"
 	expected_output=$(cat <<-EOM
 		Loading required manifests...
-		Downloading packs for:
-		 - test-bundle1
-		 - test-bundle2
-		Finishing packs extraction...
-		No extra files need to be downloaded
-		Installing bundle(s) files...
+		No packs need to be downloaded
+		Starting download of remaining update content. This may take a while...
+		Staging file content
```

Comment:
should we add "..." at the end of the actions written in present participle: "Staging file content" and "Installing files"? We do that with most other phrases that use verbs with that form (with some exceptions like "Calling post-update helper scripts").

Reply:
or perhaps we should remove the "..." from the other messages?

Reply:
Maybe I'm being too picky haha

Reply:
no, you're not. I think we should define that. No strong opinion though. Should we add it?

Reply:
Yeah, let's go with adding the "..." for phrases in present participle for now, if we ended up having too many "..." and that cause the output to look ugly we can reconsider in a future PR.